rel/overlay/etc/default.ini

```diff
@@ -267,11 +268,11 @@ credentials = false
 ; List of origins separated by a comma, * means accept all
 ; Origins must include the scheme: http://example.com
 ; You can't set origins: * and credentials = true at the same time.
-;origins = *
+origins = *
 ; List of accepted headers separated by a comma
-; headers =
+headers = authorization,content-type
 ; List of accepted methods
-; methods =
+methods = GET,POST
```

Comment:
Again, these are unrelated changes, unless CORS support is mandatory for JWT, and only if we enable JWT by default. (I'm now less emphatic about that myself.)